pkg/msk.go

```diff
@@ -78,7 +78,8 @@ func (e *MSKExporter) addMetricFromMSKInfo(sessionIndex int, clusters []*kafka.C
 			}
 			e.cache.AddMetric(prometheus.MustNewConstMetric(MSKInfos, prometheus.GaugeValue, 1, region, clusterName, mskVersion, eolDate, eolStatus))
 		} else {
-			level.Error(e.logger).Log("msg", "EOL information not found for MSK version", "version", mskVersion)
+			level.Info(e.logger).Log("msg", "EOL information not found for MSK version %s, setting status to 'green'", mskVersion)
+			e.cache.AddMetric(prometheus.MustNewConstMetric(MSKInfos, prometheus.GaugeValue, 1, region, clusterName, mskVersion, "no-eol-date", "green"))
```

Comment:
What if "green" is not specified? Maybe it's better to skip unknown versions at all?!?

Reply:
Maybe it could have an "unknown" status? It would be nice to have all cluster versions included for full visibility.

Reply:
"unknown" sounds perfect to me!